lib/style/_open-sans.scss

```diff
@@ -15,16 +15,22 @@
  * limitations under the License.
  */
 
+@font-face {
+  font-family: 'Open Sans';
+  font-weight: 400;
+  font-style: normal;
+  font-display: swap;
+  src: local('Open Sans'), local('Open Sans Regular'), local('OpenSans-Regular'), local('Open-Sans-regular'),
+  url('../../lib/fonts/Open-Sans-regular/Open-Sans-regular.woff2') format('woff2'),
+  url('../../lib/fonts/Open-Sans-regular/Open-Sans-regular.woff') format('woff');
+}
+
 @font-face {
   font-family: 'Open Sans';
   font-weight: 600;
   font-style: normal;
-  src: url('/fonts/Open-Sans-600/Open-Sans-600.eot');
-  src: url('/fonts/Open-Sans-600/Open-Sans-600.eot?#iefix') format('embedded-opentype'),
-  local('Open Sans Semibold'),
-  local('Open-Sans-600'),
-  url('/fonts/Open-Sans-600/Open-Sans-600.woff2') format('woff2'),
-  url('/fonts/Open-Sans-600/Open-Sans-600.woff') format('woff'),
-  url('/fonts/Open-Sans-600/Open-Sans-600.ttf') format('truetype'),
-  url('/fonts/Open-Sans-600/Open-Sans-600.svg#OpenSans') format('svg');
+  font-display: swap;
+  src: local('Open Sans SemiBold'), local('OpenSans-SemiBold'), local('Open Sans Semibold'), local('Open-Sans-600'),
+  url('../../lib/fonts/Open-Sans-600/Open-Sans-600.woff2') format('woff2'),
```

Comment:
Why we can drop other formats?

Reply:
I don't think we have to support legacy formats. Support for woff looks pretty solid https://caniuse.com/?search=woff. On Allegro we support just woff and woff2.